ITEM_en.tsv

```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
 ITEM_20150612_005304	Green Kokateu Bone
 ITEM_20150612_005305	A Green Kakatue bons is very hard so it is mainly used to make the arrowheads.
 ITEM_20150612_005306	Colson Signal Bell
-ITEM_20150612_005307	A tool send the signal to his companion.
+ITEM_20150612_005307	A tool Colson used to send a signal to his companion.
 ITEM_20150612_005308	Scorpion Poison Sample
 ITEM_20150612_005309	An animal poison sample obtained from the scorpion.
 ITEM_20150612_005310	Thorn Poison Sample
 ITEM_20150612_005311	A poison sample obtained from the thorn tree stump.
 ITEM_20150612_005312	Light Well Poison Sample
-ITEM_20150612_005313	A land of poison sample obtained from the light well.
+ITEM_20150612_005313	Apoison sample obtained from the light well.
 ITEM_20150612_005314	Water Bowl
-ITEM_20150612_005315	A tool scoop the poison of light well
+ITEM_20150612_005315	A tool to scoop out the poison in the light well
 ITEM_20150612_005316	Dusk Grass
-ITEM_20150612_005317	A grass that Nethercaffe like very well.
+ITEM_20150612_005317	A grass that Nethercaffe particularly enjoy.
 ITEM_20150612_005318	Chalk for display
 ITEM_20150612_005319	A tool that marks on the wall or floor.
 ITEM_20150612_005320	Wild Honey
-ITEM_20150612_005321	A honey gathered by the wild bees. It is not suitable for eating but cure purpose is very useful.
+ITEM_20150612_005321	A honey gathered by the wild bees. It is not suitable for eating but its recovery purposes are very useful.
```

Comment:
Honey gathered by wild bees. It's not suitable for consumption, but its recovery effects are great.